python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -758,7 +775,14 @@ def _resume_tick(
         submit_threadpool_executor=submit_threadpool_executor,
         sensor_debug_crash_flags=sensor_debug_crash_flags,
     )
-    context.update_state(TickStatus.SUCCESS, cursor=context._tick.cursor)  # noqa # TODO
+    if context.tick.tick_data.user_interrupted:
+        context.update_state(
+            TickStatus.SKIPPED,
+            cursor=context.tick.cursor,
+            skip_reason="Sensor manually stopped mid-iteration.",
+        )
+    else:
+        context.update_state(TickStatus.SUCCESS, cursor=context.tick.cursor)  # TODO
```

Comment:
past me was terrible and did not explain this TODO, but I *think* it was just referencing the fact that we were accessing a private variable, which after your change is no longer necessary, so I think we can remove that comment